gossip/src/push_active_set.rs

```diff
@@ -212,21 +409,20 @@ mod tests {
     }
 
     #[test]
-    fn test_push_active_set() {
+    fn test_push_active_set_static_weighting() {
         const CLUSTER_SIZE: usize = 117;
```

Comment:
Unrelated to this PR, but should we not make this ~4000 for this test to be more representative?